spec/unit/inputtext_spec.lua

```diff
@@ -0,0 +1,57 @@
+describe("InputText widget module", function()
+    local InputText
+    setup(function()
+        require("commonrequire")
+        InputText = require("ui/widget/inputtext")
+    end)
+
+    describe("addChar()", function()
+        -- thanks to https://stackoverflow.com/a/32660766/2470572
+        local function equals(o1, o2, ignore_mt)
```

Comment:
I should probably stick this in `setup()` instead since other unit tests on this module would also want to use it.